server/controllers/personal/personalController.test.ts

```diff
@@ -2090,8 +2101,66 @@ describe('PersonalController', () => {
   describe('Diet and food allergies', () => {
     describe('Edit', () => {
       const action = async (req: any, response: any) => controller.dietAndFoodAllergies().edit(req, response, () => {})
+      const editOptions = ({ selections }: { selections: ReferenceDataIdSelection[] }) => ({
```

Comment:
The testing around this needed improving so taking the opportunity to do it here